src/applications/check-in/utils/navigation/day-of/index.js

```diff
@@ -31,48 +34,58 @@ const CHECK_IN_FORM_PAGES = Object.freeze([
     order: 0,
   },
   {
-    url: URLS.DEMOGRAPHICS,
+    url: URLS.LOADING,
     order: 1,
   },
   {
-    url: URLS.NEXT_OF_KIN,
+    url: URLS.DEMOGRAPHICS,
     order: 2,
   },
   {
-    url: URLS.EMERGENCY_CONTACT,
+    url: URLS.NEXT_OF_KIN,
     order: 3,
   },
   {
-    url: URLS.UPDATE_INSURANCE,
+    url: URLS.EMERGENCY_CONTACT,
```

Comment:
Not part of this PR, but a heads up, that Emergency Contact should come before next of kin

Reply:
good to know, I'm going to rebase and catch up after I get all the tests sorted.